### PR TITLE
add markdown link checks

### PR DESCRIPTION
**Pull request by matancarmeli7**
Signed-off-by: matancarmeli7 <matan.carmeli7@gmail.com>

### CODE REVIEW

**Review by oriyarde — DISMISSED** [review]
LGTM

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Run link check
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-verbose-mode: 'yes'
+        check-modified-files-only: 'yes'
```
Comment (ArbelNathan) [inline .github/workflows/docs.yaml:19]: image

Comment (matancarmeli7) [reply .github/workflows/docs.yaml]: done

File: .github/workflows/docs.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Check Markdown links
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
```
Comment (ArbelNathan) [inline .github/workflows/docs.yaml:7]: we have `master`..

Comment (matancarmeli7) [reply .github/workflows/docs.yaml]: done

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**